modules/andes-core/client/src/main/java/org/wso2/andes/client/AMQConnectionDelegate_8_0.java

```diff
@@ -116,6 +116,11 @@ public ProtocolVersion makeBrokerConnection(BrokerDetails brokerDetail) throws A
             settings.setTrustStoreCertType(sslConfig.getCertType());
         }
 
+        if(settings.getKeyStorePath() == null && settings.getKeyStorePassword() ==null && settings
+                .getTrustStorePath() == null && settings.getTrustStorePassword() == null){
+              settings.loadSSLConfigFromSysConfig();
```

Comment:
WE can't move this block above sslCOnfig null check , because  if we do so, settings will be override with null values